google/cloud/sql/connector/connector.py

```diff
@@ -91,6 +91,7 @@ def __init__(
         loop: Optional[asyncio.AbstractEventLoop] = None,
         quota_project: Optional[str] = None,
         sqladmin_api_endpoint: str = "https://sqladmin.googleapis.com",
+        custom_user_agent: Optional[str] = None,
```

Comment:
nit: the fact that the user is configuring makes it "custom", redundant to have it in the var name
```suggestion
        user_agent: Optional[str] = None,
```

Reply:
Fixed.